drivers/sensor/lps22hb/Kconfig

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright (c) 2017 Linaro Limited
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig LPS22HB
+	bool
+	prompt "LPS22HB pressure and temperature"
+	depends on SENSOR && I2C
+	default n
+	help
+	  Enable driver for LPS22HB I2C-based pressure and temperature
+	  sensor.
+
+config LPS22HB_DEV_NAME
+	string
+	prompt "Device name"
+	default "LPS22HB"
+	depends on LPS22HB
+	help
+	 Device name used for LPS22HB sensor identification.
+
+config LPS22HB_I2C_ADDR
+	hex
+	prompt "I2C address"
+	depends on LPS22HB
+	default "0x5D"
```

Comment:
Since it's a hex config, I don't think you need the "" around the value.
You could also add: range 0x5C 0x5D.